code/src/index.css

```diff
@@ -5,9 +5,85 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  font-size: 15px;
 }
 
 code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.form-container, .thought-container {
+  border: 2px solid #797878;
+  margin: 20px;
+  box-shadow: 6px 6px #060606;
+  padding: 10px;
+  font-family: source-code-pro;
+  margin-left: auto;
+  margin-right: auto;
+  width: 280px;
+}
+
+.form-container {
+  background-color: #F2F0F0;
+  letter-spacing: 0.5px;
+}
+
+.main-title {
+  font-weight: 600;
+}
+
+.input-style {
+  margin-top: 10px;
+  border-color: #DEDEDE;
+  border-style: solid;
+  resize: none;
+  width: 97%;
+}
+
+.submit-button {
+  border-radius: 20px;
+  margin-top: 10px;
+  background-color: #F0ACAC;
+  padding: 10px;
+  border: none;
+  cursor: pointer;
+  outline: none;
+}
+
+.thought-container {
```

Comment:
when someone sends a very long word (not sure in what case that would happen.. well in finish there might be some longer ones 🙃) it overwrites the container and the cards also aren't centered anymore. I solved that by adding " overflow-wrap: break-word" to the container. Maybe you could give it a try. If it works it would make your happy thoughts app even happier! 🤩  🦄